### PR TITLE
Updated description for SSD models

### PR DESCRIPTION
**Pull request by Anna-Mironova**

### CODE REVIEW

File: models/public/ssd300/ssd300.md [hunk 1]
```diff
@@ -23,12 +23,12 @@ See [here](https://github.com/weiliu89/caffe/tree/ssd).
 
 ## Accuracy
 
+The accuracy results were obtained on test data from VOC2012 dataset.
```
Comment (eaidova) [inline models/public/ssd300/ssd300.md:26]:
```suggestion
The accuracy results were obtained on validation data from VOC2012 dataset.
```
test part of dataset is avaliable only on the server

File: models/public/ssd512/ssd512.md [hunk 2]
```diff
@@ -23,12 +23,12 @@ See [here](https://github.com/weiliu89/caffe/tree/ssd).
 
 ## Accuracy
 
+The accuracy results were obtained on test data from VOC2012 dataset.
```
Comment (eaidova) [inline models/public/ssd512/ssd512.md:26]:
```suggestion
The accuracy results were obtained on validation data from VOC2012 dataset.
```

File: models/public/ssd300/ssd300.md (hunk 1 above)
Comment (Anna-Mironova) [reply models/public/ssd300/ssd300.md]: Updated

File: models/public/ssd512/ssd512.md (hunk 2 above)
Comment (Anna-Mironova) [reply models/public/ssd512/ssd512.md]: Updated

**Review by eaidova — APPROVED** [review]

**Pull request merged**